### PR TITLE
Wolicheng.WoMic version 4.7

### PR DESCRIPTION
**Pull request by AlexZorzi**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/18939)

### CODE REVIEW

File: manifests/w/Wolicheng/WoMic/4.7/Wolicheng.WoMic.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Wolicheng.WoMic
+PackageVersion: 4.7
+PackageLocale: en-US
+Publisher: Wolicheng Tech
+PackageName: Wo Mic
```
Comment (OfficialEsco) [inline manifests/w/Wolicheng/WoMic/4.7/Wolicheng.WoMic.locale.en-US.yaml:8]:
```suggestion
PackageName: Wo Mic Client
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Hmm, would it be difficult to contact the developer and request them to add DisplayVersion and Publisher so it shows in Windows?
image

**Comment by AlexZorzi** [comment]
@OfficialEsco i doubt they will DisplayVersion and Publisher beacause it dosent seems like devs are very active on the project, but i tried to email them anyway lets see

File: manifests/w/Wolicheng/WoMic/4.7/Wolicheng.WoMic.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Wolicheng.WoMic
+PackageVersion: 4.7
+PackageLocale: en-US
+Publisher: Wolicheng Tech
+PackageName: WO Mic Client
+License: Proprietary
+ShortDescription: Wo Mic client
```
Comment (ItzLevvie) [inline manifests/w/Wolicheng/WoMic/4.7/Wolicheng.WoMic.locale.en-US.yaml:10]:
```suggestion
ShortDescription: WO Mic can turn your phone to be a microphone for your computer.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**